src/Uri.php

```diff
@@ -178,6 +160,6 @@ public function isValid()
      */
     public function isAbsolute()
     {
-        return $this->scheme !== null;
+        return ! empty($this->scheme);
```

Comment:
`! empty()` here seems wrong, considering `$this->scheme` is `string|null`

Reply:
But an `''` is a possible string value, and that value would be considered a non-absolute URI. Hence the check for `! empty()`.